docarray/typing/tensor/abstract_tensor.py

```diff
@@ -13,9 +13,48 @@
 ShapeT = TypeVar('ShapeT')
 
 
+class _ParametrizedMeta(type):
```

Comment:
not sure that type is needed here

Reply:
Not sure if it is strictly needed bc in the end the "final" metaclass will only inherit from this one, but I think if we wanted to use `_ParametrizedMeta` directly as a metaclass we would need it. So I think it is good practice for all metclasses to do it